xml/cap_admin_create_admin.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.create-admin-user"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Creating Admin Users</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  This chapter provides an overview on how to create additional
+  administrators for your &cap; cluster.
+ </para>
+ <sect1 xml:id="sec.cap.create-admin-prereqs">
+  <title>Prerequisites</title>
+
+  <para>
+   The following prerequisites are required in order to create
+   additional &cap; cluster administrators:
+  </para>
+
+  <itemizedlist>
+   <listitem>
+    <para>
+     The Cloud Foundry command line interface (&cfcli;). (See
+     <xref linkend="sec.cap.cf-cli"/>.)
+    </para>
+   </listitem>
+   &uaac-prereq;
+  </itemizedlist>
+
+ </sect1>
+ <sect1 xml:id="sec.cap.create-admin-procedure">
+  <title>Creating an example &cap; cluster administrator</title>
+  <para>
+   The following example demonstrates the steps required to create a new
+   administrator user for you &cap; cluster. Note that creating administrator
```

Comment:
`for you &cap;` :arrow_right: `for you_r_ &cap;`

Reply:
Thanks, fixed in https://github.com/SUSE/doc-cap/commit/0c37e383b939a98a8e48214fa5154d16fb266682